test/unit/core/diffusion_test.cc

```diff
@@ -974,8 +974,8 @@ TEST(DiffusionTest, EulerDirichletBoundaries) {
   double simulation_time_step{0.1};
   auto set_param = [](auto* param) {
     param->bound_space = Param::BoundSpaceMode::kClosed;
-    param->min_bound = -100;
-    param->max_bound = 100;
+    param->min_bound = -50;
+    param->max_bound = 50;
```

Comment:
This modification looks good to me. The test verifies that with dirichlet = 1, at some point all voxels converge to 1. Making the domain smaller speeds up the test.

Reply:
Looks good to me, the logic behind the test wasn't altered.